packages/styles/src/core.scss

```diff
@@ -2,7 +2,4 @@
 @forward './variables';
 @forward './mixins';
 
-@use './lic/bootstrap-license';
```

Comment:
Don't we need to import the bootstrap-license anymore?

Reply:
We do have to include it, reverted the change: https://getbootstrap.com/docs/5.2/about/license/#it-requires-you-to